Source/GSSocketStream.h

```diff
@@ -30,7 +30,7 @@
 #import "GSNetwork.h"
 
 typedef	union {
-  struct sockaddr	s;
+  struct sockaddr_storage	s;
```

Comment:
During development GNUStep was crashing with a memory exception because sockaddr isn't big enough to store an IPv6 address and information. Sockaddr_storage is the new standard for storing IP addresses in WIN32 applications https://learn.microsoft.com/en-us/windows/win32/winsock/sockaddr-2

Reply:
Ok understood. And we are sure this will only apply to Windows and not other platforms?

Reply:
Actually ok I have read the spec on this and I think it should be fine.